composer.json

```diff
@@ -25,14 +25,15 @@
     "symfony/config": "^2.2|^3.0|^4.0",
     "goetas-webservices/xsd-reader": "^0.2|^0.3.1",
     "doctrine/inflector": "^1.0",
-    "zendframework/zend-code": "~2.3",
+    "zendframework/zend-code": "~2.3|^3.0.3",
```

Comment:
what about getting rid to the v2.3?

Reply:
Yes I can do that. Didn't want to alter the deps unless it was necessary.